util/Migrator/DbScripts/2023-03-07_00_CipherKeyUpdate.sql

```diff
@@ -0,0 +1,440 @@
+-- Add Key Column
+IF COL_LENGTH('[dbo].[Cipher]', 'Key') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Cipher] ADD [Key] VARCHAR (MAX) NULL;
+END
+GO
+
+-- Add ForceKeyRotation
+IF COL_LENGTH('[dbo].[Cipher]', 'ForceKeyRotation') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Cipher] ADD [ForceKeyRotation] BIT NULL;
+    UPDATE [dbo].[Cipher] SET [ForceKeyRotation] = 0;
+    ALTER TABLE [dbo].[Cipher] ALTER COLUMN [ForceKeyRotation] BIT NOT NULL;
```

Comment:
ℹ️ Let's do as @Hinton described -- more performant.

Reply:
Yeah, I confirmed it's more performant. Thanks.